src/main/java/com/google/cloud/tools/maven/CloudSdkAppEngineFactory.java

```diff
@@ -154,8 +157,16 @@ private AppEngineDevServer createDevServerForVersion(
    */
   public AppEngineDevServer devServerRunAsync(
       int startSuccessTimeout, SupportedDevServerVersion version) {
+    File logFile = new File(mojo.mavenProject.getBuild().getDirectory(), "dev_appserver.out");
```

Comment:
Should the log output directory be configurable?

Reply:
It doesn't look like it is in the gradle plugin.

Reply:
Ah okay, it looks like in the gradle plugin, it puts the out file in a dev-appserver-out directory.

https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/156/files#diff-f80e036458fbefa4397a6bedcdc45897R45